R/wflow_toc.R

```diff
@@ -0,0 +1,45 @@
+#' Add a table of content on the clipboard
+#'
+#' \code{wfow_toc} add a table of content on the clipboard. The users can paste it
+#' in the \code{index.Rmd} document. The table of content includes all \code{html} document
+#' in the \code{docs} directory the users want to display.
+#'
+#' @param gh_url character. The Github Pages root directory.
+#' @param proj_url character. The project name.
+#'
+#' @return text character. The list of table of content in the markdown syntax.
+#' @import fs
+#' @import glue
+#' @import dplyr
+#' @import purrr
+#' @import readr
+#' @import tidyr
+#' @import clipr
+#' @import here
+#' @import rmarkdown
+#' @import tibble
+#' @export
+
+wflow_toc <- function(analysis_dir = "analysis", docs_dir = "docs") {
+  s <- wflow_status()
+  s$status %>%
+    tibble::rownames_to_column(var = 'rmd_path') %>%
```

Comment:
Do you know how to perform these steps without relying on the tidyverse functions? My package doesn't currently use these packages, and I don't want to add a lot of new dependencies for just this one function. If needed, I can translate these to base R commands.

Reply:
Yes, I agree with you. I will try it first.

Reply:
Yes, I agree with you. I will try it first.

Reply:
I edit it, but I don't find a good substitute for `dplyr::*join` function, do you have any recommendations?
here is my handle, https://github.com/jdblischak/workflowr/pull/155/commits/cdbb9c3fcde89164564e6002b97bc93d37f1563a